S3GenericMethods.R

```diff
@@ -0,0 +1,115 @@
+
+#' Print function for safe test designs for two proportions
+#'
+#' @param x Design for a safe test
+#' @param ... Other arguments for the generic print function
+#'
+#' @return No return value
+#' @export
+#'
+print.safe2x2_result <- function(x, ...){
+  cat("\n")
+  cat(paste("       ", "Safe Design for Test of Two Proportions", "\n"))

```

Comment:
I figured out that you don't use paste within a cat, just combine the strings:

cat("       ", "Safe Design for Test of Two Proportions", "\n", sep=" ")